segmentstore/storage/src/main/java/io/pravega/segmentstore/storage/SegmentRollingPolicy.java

```diff
@@ -10,22 +10,23 @@
 package io.pravega.segmentstore.storage;
 
 import com.google.common.base.Preconditions;
+import io.pravega.common.io.serialization.RevisionDataOutput;
 import lombok.Getter;
 
 /**
  * A generic rolling policy that can be applied to any Storage unit.
  */
 public final class SegmentRollingPolicy {
     /**
-     * The max allowed value for 61 bit signed number which is 2,305,843,009,213,693,952.
+     * The max allowed value for segment size.
      */
-    public static final long MAX_62_BIT_SIGNED_NUMBER = 1L << 61;
+    public static final long MAX_SGEMENT_LENGTH = RevisionDataOutput.COMPACT_LONG_MAX;
```

Comment:
1. typo
2. This is "max chunk length", not max segment length. Segments can have a `Long.MAX_VALUE` length.

Reply:
fixed

Reply:
https://github.com/pravega/pravega/pull/4686/commits/1fd6faf4373ba543b9c590a545d9786a89fd1ac8